draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockMenuItem.module.scss

```diff
@@ -49,8 +49,7 @@
     }
   }
 
-  &:focus-visible,
-  &.focus-visible {
+  &:focus-visible {
```

Comment:
Do we need to move `outline: none` to `&:focus {}`?

Reply:
Yep, I've done that now